docs/config/lua/config/quote_dropped_files.md

```diff
@@ -0,0 +1,27 @@
+## quote_dropped_files = "SpacesOnly"
+
+*Since: nightly builds only*
+
+Controls how file names are quoted (or not) when dragging and dropping.
+There are three possible values:
+
+* `"None"` - no quoting is performed, the file name is passed through as-is.
+* `"SpacesOnly"` - backslash-escape only spaces, leaving all other characters as-is.  This is the default.
+* `"Posix"` - use POSIX style shell work escaping.
+
+For example
+
+|`quote_dropped_files` |file name    |quoted result  |
+|----------------------|-------------|---------------|
+|`"None"`              |`hello world`|`hello world`  |
+|`"SpacesOnly"`        |`hello world`|`hello\ world` |
+|`"Posix"`             |`hello world`|`"hello world"`|
+
+Files drag and drop support is a platform dependent feature
```

Comment:
```suggestion
Drag and drop support for files is a platform dependent feature
```

Adding a apostrophe after "Files" is enough, but this one rings better in this case? I am not a native speaker, btw.